src/ui/components/shared/Login/Login.tsx

```diff
@@ -136,19 +136,21 @@ function SocialLogin({
   return (
     <div className="space-y-6">
       <LoginMessaging />
-      <PrimaryLgButton
-        color="blue"
-        onClick={() => onLogin("google-oauth2")}
+      <Button
         className="w-full justify-center"
+        onClick={() => onLogin("google-oauth2")}
+        size="large"
       >
         Sign in with Google
-      </PrimaryLgButton>
-      <button
```

Comment:
This was a non-standard button component beside of a standard one. Probably not great?